src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+
+  &--1 :nth-child(-n+1),
+  &--2 :nth-child(-n+2),
+  &--3 :nth-child(-n+3),
+  &--4 :nth-child(-n+4),
+  &--5 :nth-child(-n+5) {
+    background-image: url(../images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
```

Comment:
pay attention on this point from checklist
![Screenshot from 2023-08-13 18-09-48](https://github.com/mate-academy/layout_catalog/assets/33029864/b734dcb3-66c8-4d8e-a017-e382b41c763e)